services/tenant-management-service/src/providers/idp/types.ts

```diff
@@ -0,0 +1,4 @@
+export type Auth0Response = {
```

Comment:
as we are using a commong resp IdpResp={authId:string}, we can remove this Auth)response